setup.py

```diff
@@ -52,6 +52,6 @@ def load_requirements(*requirements_paths):
         'Programming Language :: Python',
     ],
     packages=PACKAGES,
-    install_requires=load_requirements('requirements.txt'),
+    install_requires=load_requirements('requirements.txt', 'tox-requirements.txt'),
```

Comment:
Not only that, but `tox-requirements.txt` could rightfully be called `django-requirement.txt`, since that's all it's controlling here :)

I'll take a look at this other PR to see if I can make this cleaner.